### ISSUE TITLE
angle brackets in html attrs break tokenizer

### ISSUE THREAD

**Issue by zavr-1** [leeoniya/dropcss#37]
Hi man I've been trying to compile some css, and what happened is that I accidentally had this:

```html
const html = `
<body>
  <li heading="<em>your-signature-here</em>">
  </li>

  <h4 id="<em>your-signature-here</em>" class="mt-2"><em>Your Signature Here</em>:</h4>
</body>`
```
although it's probably wrong to put tags in attributes (it was autogenerated), the issue is that the error from `dropcss` does not really give much info 
```
TypeError: Cannot read property 'parentNode' of null
    at build (/Users/zavr/adc/splendid/node_modules/dropcss/dist/dropcss.cjs.js:154:17)
```
is it possible to maybe give more debug info? 
also maybe tags in attributes should result in some kind of warning? thanks.

**Comment by leeoniya** [comment]
> is it possible to maybe give more debug info?

the parser is quite rudimentary, so i don't think it can provide much feedback in this case; by the time it gets to the tree builder it has already been improperly tokenized. however, i think i can improve the parser a bit to properly handle angle brackets in attrs (which is the actual bug here).

> although it's probably wrong to put tags in attributes

and `li`s outside of a list ;)

> also maybe tags in attributes should result in some kind of warning?

if this is something you'd like to prevent in your code (unrelated to DropCSS), then i think it would make the most sense to write a simple regex-based test to check your markup.

**Comment by leeoniya** [comment]
this is fixed in v1.0.15
